cmd/erda-server/conf/i18n/i18n.yaml

```diff
@@ -21,6 +21,12 @@ zh:
   APPLICATION: 应用
   ErrAddMemberOwner: "添加所有者失败，数量超过上限"
 
+  # audit error
+  ErrInvalidOrg: "OrgID 非法，可能是不存在orgID或者多个orgID"
```

Comment:
Add spaces between Chinese and English text.
And use same case for orgID.

Reply:
has been fixed